github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1906,3 +1906,46 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: false
+    annotations:
+      fork-per-release: "true"
+    cluster: ibm-prow-jobs
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 1h0m0s
+    labels:
+      preset-bazel-cache: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-podman-in-container-enabled: "true"
```

Comment:
this preset needs to be changed if you want to try the legacy image.

`image: quay.io/kubevirtci/golang-legacy:v20220810-a8f2e6c`

requires 

`preset-dind-enabled: "true"`

Reply:
I added preset-dind-enabled: "true".

> but we should try to get this working with the podman setup.

not sure what else to try, I tried every syntax I could think of so I need help (:

Reply:
@brianmcarey when running manually this is the output: http://pastebin.test.redhat.com/1103045.